lib/tapioca/generators.rb

```diff
@@ -0,0 +1,5 @@
+# typed: true
```

Comment:
Moved to a top-level `generators` namespace to make for easier updates in the future. Can be updated in the future to dynamically pull in all generators.